ipaullly/sendIT
# [#161796710] Create endpoint to register user

#### What does this PR do?
Creates an endpoint that allows a user to register an account with unique email and password
#### Description of Task to be completed?
A user can send a POST request to register an account and receives a success message with 201 status code
#### How should this be manually tested?
clone repo into a local directory
```git clone --single-branch -b ft-POST-register-161796710 git@github.com:ipaullly/sendIT.git```
cd into the sendIT directory
activate virtual environment
```source venv/bin/activate```
install dependencies
```pip install requirements.txt```
in the home directory execute the test runner to check whether all tests pass
```pytest```
power up POSTMAN and send a JSON body with email and password to Registration API. sending the same details again will prompt a message advising the user to log into their account instead.
#### What are the relevant pivotal tracker stories?
#28
